components/header.js

```diff
@@ -46,9 +41,14 @@ export default props => {
       external: false,
     },
     {
+      title: 'Events',
+      path: '/events',
+      external: false,
+    },
+    {
       title: 'Blog',
       path: 'https://medium.com/coderplex',
-      external: true,
+      external: false,
```

Comment:
Why is this changed to `false`?

Reply:
opening blog feels its taking away from web site, i feel it should be continuous